### PR TITLE
ownCloud is no backup, ext storage doesn't sync

### PR DESCRIPTION
**Pull request by jospoortvliet**
@phil-davis your input would be very welcome!

### CODE REVIEW

File: page-faq.php [hunk 1]
```diff
@@ -68,6 +68,9 @@
...
 <p>All this is free, as ownCloud is available under an '<em>open source license</em>', which gives you the right to examine, share <strong>and</strong> modify it. ownCloud is developed by an <a href="/contact">international community</a> of both paid and volunteer contributors and <a href="/contribute">you can get involved, too</a>.</p>
+<h3>Things ownCloud does not do</h3>
+<p>Note that ownCloud is NOT a backup tool: the desktop client, for example, will synchronize the contents of a local folder with the server. That works for file modifications and additions, but also deletions. If you delete a file on the server, it gets deleted on the desktop you installed the client on! ownCloud does have a <strong>trasn</strong> and <strong>Versions</strong> support, which means deleted or modified files can be brought back, but this can be disabled and depending on the space usage settings, these older files do get deleted over time. You should always backup your ownCloud server!</p>
```
Comment (RealRancor) [inline page-faq.php]: <strong>trasn</strong>

-> <strong>trash</strong> or maybe even <strong>Trash</strong>

File: page-faq.php [hunk 2]
```diff
@@ -68,6 +68,9 @@
...
+<p>Similarly, note that External Storage and the Server to Server sharing which ownCloud supports, connects ownCloud to an external storage like FTP or Samba, or to another ownCloud server. Files will show up in your web interface and can be synced with the client. However, they are <strong>not replicated</strong> on your ownCloud server! They are merely fetched whenever you request them in the web interface. Once the external storage disappears or other ownCloud server retracts your access, you can no longer access the files! Make a copy if you want to have them on your own ownCloud server.</p>
```
Comment (phil-davis) [inline page-faq.php]: There is still 1 "or other ownCloud" here - change to "or another ownCloud".

File: page-faq.php (hunk 1 above)
Comment (phil-davis) [inline page-faq.php]: In addition, no need to say "a Trash and Versions support" - just "Trash and Versions support".

File: page-faq.php (hunk 2 above)
Comment (phil-davis) [inline page-faq.php]: "it gets deleted on the desktop you installed the client on!"
maybe put even more, like:
"it gets deleted on all devices that you installed the client on!"
and should it also spell out the reverse by adding a sentence like:
"If you delete a file on the client, it gets deleted on the server and then deleted on other clients that sync to that were syncing that file!"
Maybe that makes the whole thing too long, and people will not read it?

**Comment by jospoortvliet** [comment]
Thanks for the feedback! Merging...

**Pull request merged**